### ISSUE TITLE
[BUG]: Some kind of mipmapping bug in Ratchet & Clank

### ISSUE THREAD

**Issue by EvilPlayer1** [PCSX2/pcsx2#11578]
### Describe the Bug

After last modification of mipmapping Ratchet & Clank works worse now. Some textures have artefacts and the image became more blurry than with old basic mipmapping, because FOV was decreased. As an example I'll take Novalis. These things can be seen in the beginning.
Mipmapping is on:
![Screenshot 17-07-2024 20-08-11](https://github.com/user-attachments/assets/556493a1-7153-46bd-814e-52fb05c8540f)
Mipmapping is off:
![Screenshot 17-07-2024 20-09-35](https://github.com/user-attachments/assets/e4c1578a-277b-4fca-be1b-a932597aeff6)
Native:
![Screenshot 17-07-2024 20-12-37](https://github.com/user-attachments/assets/8d15c31e-5ed9-4dd2-bc8a-8b3744dbec87)
I'm using the texture pack and because of new mipmapping all textures can't be upscaled properly

### Reproduction Steps

Go to Novalis with mipmapping enabled

### PCSX2 Revision

v2.0.3

### Operating System

Windows 11

### CPU

i7-11700

### GPU

GTX 1660 SUPER

**Comment by Mrlinkwii** [comment]
Ratchet & Clank series  has mipmapping issues see  #5008

**Comment by Mrlinkwii** [comment]
are you using a texture pack @EvilPlayer1 ?

**Comment by EvilPlayer1** [comment]
Yes. that's why I created this thread. With old mipmapping everything was upscaled properly, but now it works bad. R&C 2 is ok, but there's visible drawing line, which makes objects more detailed while getting closer to them. I don't remember that it was so noticeable earlier.

**Comment by lextra2** [comment]
Just to clarify

>Mipmapping On

Is incorrect because it MipMaps **all** textures

>Mipmapping Off

Is incorrect because some textures need to be MipMapped

>Native

Is incorrect because Software renderer has bugs

tldr: Pick your poison

I think https://github.com/PCSX2/pcsx2/pull/11449 could have fixed it. Oh well...

**Comment by Loy2up** [comment]
Worth mentioning these bugs can all be reproduced on the original PS2 hardware, albeit less frequently.
